code/index.html

```diff
@@ -2,21 +2,150 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name = "viewport" content="width = device- width, 
+    initial-scale = 1.0">
+    
   </head>
+  
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <div class="sticky">
+
+        <img src="logo.svg" alt="TECHNIGO LOGO"/>
+        <h2>Happy New Year!</h2>
+      
+        <nav>
+            <a href="#home">HOME</a>
+            <a href="#modern">MODERN</a>
+            <a href="#old">OLD</a>
+            <a href="#asia">ASIA</a>
+            <a href="#america">AMERICA</a>
+            <a href="#australia">AUSTRALIA</a>
+
+        </nav>
+
+      </div>  
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+    
+
+    <section class= "big-news-container">
+            
+      <img src = "./images/firework1.jpg" alt="BIG NEWS" />
+      <div class = "big-news-child">
+          	<p class = "main-news"><a name = "home">New Year celebrations around the world 
```

Comment:
Didn't know that it could be used for the `<a>` tag, I've been looking at [W3School's](https://www.w3schools.com/tags/att_name.asp) list. But as you say, it seems to be working 👍